doc/stdenv/passthru.chapter.md

```diff
@@ -104,9 +104,9 @@ stdenv.mkDerivation (finalAttrs: {
 
 However, tests that are non-trivial will better fit into `passthru.tests` because they:
 
-- Access the package as consumers would, independently from the environment in which it was built
+- Access the package as consumers would, e.g. the environment setup up the way `mkDerivation` or `environment.systemPackages` would.
```

Comment:
I'm not sure `mkDerivation` makes sense to me here

Reply:
`s/mkDerivation/stdenv/`?